pkg/karpenter/providers/helm/helm.go

```diff
@@ -0,0 +1,111 @@
+package helm
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kris-nova/logger"
+	"gopkg.in/yaml.v1"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/repo"
+
+	"github.com/weaveworks/eksctl/pkg/karpenter/providers"
+)
+
+// Options defines options for the Helm Installer.
+type Options struct {
+	Namespace string
+}
+
+// Installer implement the HelmInstaller interface.
+type Installer struct {
+	Settings     *cli.EnvSettings
+	Getters      getter.Providers
+	ActionConfig *action.Configuration
+}
+
+// NewInstaller creates a new Helm backed Installer for repo resources.
+func NewInstaller(opts Options) (*Installer, error) {
+	settings := cli.New()
+	actionConfig := new(action.Configuration)
+	if err := actionConfig.Init(settings.RESTClientGetter(), opts.Namespace, os.Getenv("HELM_DRIVER"), logger.Debug); err != nil {
```

Comment:
what this env var?

Reply:
Oh I think I need to remove this ( I'll do it in the next PR but good call ). Defines the storage backend for helm. By default it's secrets now. I don't think we should allow the user to influence this. But maybe we will. I don't know really.

Reply:
resolved, but will fix in the coming pr